app/src/main/java/eu/darken/myperm/permissions/core/known/APerm.kt

```diff
@@ -384,15 +384,12 @@ sealed class APerm constructor(val id: Permission.Id) {
         override val groupIds: Set<PermissionGroup.Id> = grpIds(APermGrp.Connectivity)
         override val tags = setOf(ManifestDoc)
     }
-
-    object ACCESS_NETWORK_STATE : APerm("android.permission.ACCESS_NETWORK_STATE") {
-        override val iconRes: Int = R.drawable.ic_network_state_24
-        override val groupIds: Set<PermissionGroup.Id> = grpIds(APermGrp.Connectivity)
-        override val tags = setOf(ManifestDoc)
-    }
-
-    object WIFI_STATE : APerm("android.permission.ACCESS_WIFI_STATE") {
-        override val iconRes: Int = R.drawable.ic_wifi_state_24
+    
+    object REQUEST_COMPANION_USE_DATA_IN_BACKGROUND :
+        APerm("android.permission.REQUEST_COMPANION_USE_DATA_IN_BACKGROUND") {
+        override val iconRes: Int = R.drawable.ic_unlimited_data_access_24
+        override val labelRes: Int = R.string.permission_unlimited_data_access_label
+        override val descriptionRes: Int = R.string.permission_unlimited_data_access_description
```

Comment:
This does not read like it would grant unlimited data:

https://developer.android.com/reference/android/Manifest.permission#REQUEST_COMPANION_USE_DATA_IN_BACKGROUND

Are you sure the permission description about unlimited data is correct?